### PR TITLE
projects: ad9739a: add IIO Support

### PR DESCRIPTION
**Pull request by amiclaus**
Allow the user to read/write via UART using IIO Osc.

Signed-off-by: Antoniu Miclaus <antoniu.miclaus@analog.com>

### CODE REVIEW

**Review by amiclaus — COMMENTED** [review]
V2:
- fix header comment

**Review by adrimbarean — APPROVED** [review]
Looks good to me.

File: projects/ad9739a-fmc-ebz/src/app_iio.c [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+#include "irq_extra.h"
+
+/******************************************************************************/
+/************************ Variables Definitions *******************************/
+/******************************************************************************/
+static struct irq_ctrl_desc *irq_desc;
+
+struct iio_data_buffer g_read_buff = {
+	.buff = (void *)DAC_DDR_BASEADDR,
+	.size = 0xFFFFFFFF,
```
Comment (adrimbarean) [inline projects/ad9739a-fmc-ebz/src/app_iio.c:60]: nitpick: should we use a define here to indicate why this size is chosen? Something like ULONG_MAX from limits.h maybe.

Comment (amiclaus) [reply projects/ad9739a-fmc-ebz/src/app_iio.c:60]: This topic was already discussed here:
https://github.com/analogdevicesinc/no-OS/pull/838

I think this should be done in all projects in a separate commit/PR.

**Pull request merged**